packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -351,7 +363,7 @@ module.exports = {
             // its runtime that would otherwise be processed through "file" loader.
             // Also exclude `html` and `json` extensions so they get processed
             // by webpacks internal loaders.
-            exclude: [/\.(js|mjs|jsx)$/, /\.html$/, /\.json$/],
+            exclude: [/\.(js|mjs|jsx|tsx?)$/, /\.html$/, /\.json$/],
```

Comment:
I think it's more clear to write `ts|tsx` rather than `?` they have `js` and `jsx` not `jsx?` so I think the pattern should be preserved everywhere.